T3/Gui/Graph/Interaction/SymbolBrowser.cs

```diff
@@ -287,12 +293,129 @@ private void DrawMatchesList()
 
             ImGui.EndChildFrame();
 
-            DrawDescriptionPanel(itemForHelp);
-
             ImGui.PopStyleVar(2);
         }
 
-        private static void DrawDescriptionPanel(SymbolUi itemForHelp)
+        private SymbolUi DetermineItemForHelp(SymbolUi symbolUi, bool isSelected, bool isHovered)
+        {
+            if (isHovered)
+            {
+                _lastHoveredSymbolUi = symbolUi;
+                return symbolUi;
+            }
+
+            if (symbolUi != _lastHoveredSymbolUi)
+                return isSelected ? symbolUi : null;
+            
+            if (UserSettings.Config.SymbolBrowserDescriptionTimeout)
+            {
+                ExpireLastHoveredSymbolUi(symbolUi);
+            }
+
+            return symbolUi;
+
+        }
+
+        /// <summary>
+        /// A method to give some time before expiring a hovered symbol.
+        /// This is intended to allow the cursor to cross over the scroll bar
+        /// of the symbol browser so examples can be interacted with
+        /// </summary>
+        /// <param name="hoveredSymbolUi"></param>
+        async void ExpireLastHoveredSymbolUi(SymbolUi hoveredSymbolUi)
+        {
+            //if we've already started calculating this one, we can just return
```

Comment:
I know it's nitpicky, but ideally comments should be with space an upper case. 
This helps me to distinguish it from commented code

```suggestion
            // If we've already started calculating this one, we can just return
```

Reply:
Yes ive been trying to remember this, my bad!